src/raw/client.rs

```diff
@@ -819,6 +809,61 @@ impl<PdC: PdClient> Client<PdC> {
         Ok(result)
     }
 
+    #[async_recursion]
+    async fn retryable_scan(
+        &self,
+        mut scan_args: ScanInnerArgs,
+    ) -> Result<(Option<RawScanResponse>, Key)> {
+        let start_key = match scan_args.start_key {
+            None => return Ok((None, Key::EMPTY)),
+            Some(ref sk) => sk,
+        };
+        let permit = scan_args.permits.acquire().await.unwrap();
```

Comment:
`permit` seems to be not necessary as there is no multi-threading.

Reply:
Removed